pkg/routes/updates.go

```diff
@@ -454,3 +454,48 @@ func SendNotificationForDevice(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 }
+
+// ValidateUpdateResponse indicates whether or not the image can be updated
+type ValidateUpdateResponse struct {
+	UpdateValid bool `json:"UpdateValid"`
+}
+
+// PostValidateUpdate validate that images can be updated
+func PostValidateUpdate(w http.ResponseWriter, r *http.Request) {
+	services := dependencies.ServicesFromContext(r.Context())
+	account, err := common.GetAccount(r)
+	if err != nil {
+		services.Log.WithFields(log.Fields{
+			"error":   err.Error(),
+			"account": account,
+		}).Error("Error retrieving account")
+		respondWithAPIError(w, services.Log, errors.NewBadRequest(err.Error()))
+		return
+	}
+
+	var images []models.Image
+	err = json.NewDecoder(r.Body).Decode(&images)
```

Comment:
Sorry forget about this
```suggestion
	if err := readRequestJSONBody(w, r, ctxServices.Log, &images); err != nil {
		return
	}
```